src/meeting/teams/speakersObserver.ts

```
@@ -0,0 +1,522 @@
+                            borderStyle.borderColor
+                        return (
+                            hasVdiFrameClass || (isBlueish(borderColor) && borderOpacity === 1)
+                        )
+                    }
+
+                    return false
+                }
+
+                function isBlueish(color: string): boolean {
+                    // EXACT SAME AS EXTENSION
+                    color = color.toLowerCase().trim()
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)